### PR TITLE
Added sqrt BF16 FWD/BWD oneDNN kernel

### PR DESCRIPTION
**Pull request by jakpiase**
### PR types
New features

### PR changes
OPs

### Describe
Added sqrt BF16 FWD/BWD oneDNN kernel for bf16 training enabling

### CODE REVIEW

**Review by wozna — APPROVED** [review]
LGTM

**Review by lidanqing-vv — APPROVED** [review]
LGTM

**Comment by jakpiase** [comment]
Hi @Aganlengzi could you please review and merge this PR?

**Review by pawelpiotrowicz — APPROVED** [review]
LGTM

**Review by Aganlengzi — APPROVED** [review]
LGTM

File: paddle/fluid/operators/mkldnn/activation_mkldnn_op.cc [hunk 1]
```diff
@@ -293,6 +292,8 @@ REGISTER_ACTIVATION_MKLDNN_BF16_KERNEL(gelu, GeluMKLDNNFunctor,
                                        GeluMKLDNNGradFunctor);
 REGISTER_ACTIVATION_MKLDNN_BF16_KERNEL(sigmoid, SigmoidMKLDNNFunctor,
                                        SigmoidMKLDNNGradFunctor);
+REGISTER_ACTIVATION_MKLDNN_BF16_KERNEL(sqrt, SqrtMKLDNNFunctor,
+                                       SqrtMKLDNNGradFunctor);
```
Comment (lidanqing-vv) [inline paddle/fluid/operators/mkldnn/activation_mkldnn_op.cc:296]: Hi, is this PR refactoring? And add UT? Need to make spreadsheet for all PRs for release2.3RC

Comment (jakpiase) [reply paddle/fluid/operators/mkldnn/activation_mkldnn_op.cc:296]: Hi @lidanqing-intel, it's not a refactoring. I have added `sqrt` kernel for BF16 training in that PR

**Pull request merged**